src/SFA.DAS.Reservations.Web/Startup.cs

```diff
@@ -54,15 +55,14 @@ public Startup(IConfiguration configuration, IHostingEnvironment environment)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            IdentityModelEventSource.ShowPII = true;
```

Comment:
Can probably remove this, or hide it behind only being visible for local and dev?

Reply:
will discuss with you